barretenberg/cpp/src/barretenberg/vm/generated/toy_verifier.cpp

```diff
@@ -32,8 +32,8 @@ bool ToyVerifier::verify_proof(const HonkProof& proof)
     using Flavor = ToyFlavor;
     using FF = Flavor::FF;
     using Commitment = Flavor::Commitment;
-    // using Curve = Flavor::Curve;
-    // using ZeroMorph = ZeroMorphVerifier_<Curve>;
+    // using PCS = Flavor::PCS;
```

Comment:
delete?

Reply:
yeah, I suppose, didn't want to mess with the code, just kind of changed the commented stuff to compile